doc/source/parameters/cfd/volume_of_fluid.rst

```diff
@@ -202,7 +207,7 @@ The default values of the VOF parameters are given in the text box below.
 	   ... final sharpening
 
 
-* ``subsection surface tension force``: Surface tension is the tendency of a liquid to maintain the minimum possible surface area. This subsection defines parameters to ensure an accurate interface between the two phases, used when at least one phase is liquid. 
+* ``subsection surface tension force``: Surface tension is the tendency of a liquid to maintain the minimum possible surface area. This subsection defines parameters to ensure an accurate interface between the two phases, used when at least one phase is liquid. ``subsection marangoni effect`` in the ``subsection surface tension force`` enables the calculation of marangoni effect (thermocapillary effect) in simulations where the surface tension gradient (:math:`\frac{\partial \sigma}{\partial \T}`) is not equal to zero.
```

Comment:
The parameter `surface tension gradient` is not described. Maybe it could be added after the description of the  parameter `curvature filter` or in  :
```suggestion
* ``subsection surface tension force``: Surface tension is the tendency of a liquid to maintain the minimum possible surface area. This subsection defines parameters to ensure an accurate interface between the two phases, used when at least one phase is liquid. ``subsection marangoni effect`` in the ``subsection surface tension force`` enables the calculation of marangoni effect (thermocapillary effect) in simulations where the ``surface tension gradient`` (:math:`\frac{\partial \sigma}{\partial \T}`) is not equal to zero.
```